tools/utilities/refine-chkpt/CMakeLists.txt

```
@@ -8,7 +8,7 @@ target_sources(${tool_exe_name}
 
 target_include_directories(${tool_exe_name} PRIVATE
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
-target_link_libraries(${tool_exe_name} PUBLIC ${amr_wind_lib_name})
+target_link_libraries(${tool_exe_name} PUBLIC ${amr_wind_lib_name} AMReX-Hydro::amrex_hydro_api)
```

Same here. These should inherit hydro like base `amrex`. Is that not happening automatically?

```suggestion
target_link_libraries(${tool_exe_name} PUBLIC ${amr_wind_lib_name})
```

---

It misses symbols without these. I couldn't figure out how to get the inheritance correct. I'm wondering if it's because of the object library.